supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -284,7 +284,7 @@
 
 *Use it*: yes
 
-*Incorrect forms*: Grub
+*Incorrect forms*: Grub, GRUB 2, GRUB2 (Note: GRUB 2 has replaced what was formerly known as GRUB (i.e. version 0.9x), which has, in turn, become GRUB Legacy.)
```

Comment:
@jheves we usually want our incorrect forms list to be just a list of terms, and any details about why they are incorrect should go into the description field.

Would you mind moving the details about GRUB 2 replacing the former GRUB up as a sentence in the description?

Reply:
Sure thing! I'll do it asap! Thank you for the feedback, Andrea!